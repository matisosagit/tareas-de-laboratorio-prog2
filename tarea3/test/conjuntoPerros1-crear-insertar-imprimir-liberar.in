crearConjuntoPerros 4
imprimirConjuntoPerros
liberarConjuntoPerros
crearConjuntoPerros 2
insertarConjuntoPerros 1
insertarConjuntoPerros 5
insertarConjuntoPerros 1
imprimirConjuntoPerros
insertarConjuntoPerros 0
imprimirConjuntoPerros
liberarConjuntoPerros
Fin