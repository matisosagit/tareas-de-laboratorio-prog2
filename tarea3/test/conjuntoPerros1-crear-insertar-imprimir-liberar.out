1> Se creó conjunto perros con cantMax 4 exitosamente.
2> Conjunto de perros:

3> Se ejecutó 'liberarTConjuntoPerros' exitosamente.
4> Se creó conjunto perros con cantMax 2 exitosamente.
5> Se ejecutó 'insertarTConjuntoPerros con perro de id 1 exitosamente.
6> Se ejecutó 'insertarTConjuntoPerros con perro de id 5 exitosamente.
7> Se ejecutó 'insertarTConjuntoPerros con perro de id 1 exitosamente.
8> Conjunto de perros:
1
9> Se ejecutó 'insertarTConjuntoPerros con perro de id 0 exitosamente.
10> Conjunto de perros:
0 1
11> Se ejecutó 'liberarTConjuntoPerros' exitosamente.
12> Fin.
